sdks/universal-router-sdk/src/utils/routerCommands.ts

```diff
@@ -162,7 +162,7 @@ export const COMMAND_DEFINITION: { [key in CommandType]: CommandDefinition } = {
     parser: Parser.Abi,
     params: [
       { name: 'recipient', type: 'address' },
-      { name: 'amount', type: 'uint256' },
+      { name: 'amountMin', type: 'uint256' },
```

Comment:
should it not just be amount? https://github.com/Uniswap/universal-router/blob/dev/contracts/base/Dispatcher.sol#L193

Reply:
oh true

Reply:
I confused myself - I was getting an error on the calldata parser but it was wrong and this is right :D